### PR TITLE
Feature/1588 activity bar chart

### PR DESCRIPTION
**Pull request by zamuzakki**
This is PR for #1588
Activity count as % of total population

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 6 min 44 sec

**Comment by LunaAsefaw** [comment]
Just to check if the calculation is correct before it is merged. It looks like the total population is approximately 222 or 223 since the calculation seems to say 90%=200/222*100. So the implemented formula is indeed activity count/ total count *100 right?

**Comment by zamuzakki** [comment]
@LunaAsefaw The total population is 225.

File: django_project/frontend/src/containers/MainPage/Metrics/TotalCountPerActivity.tsx [hunk 1]
```diff
@@ -1,148 +1,162 @@
...
+
+    datasets.push(dataset);
   }
 
+  const data = {
+    labels: [''],
+    datasets: datasets,
+  };
+
+  const options = {
```
Comment (dimasciput) [inline django_project/frontend/src/containers/MainPage/Metrics/TotalCountPerActivity.tsx:104]: @zamuzakki, this method could limit flexibility, as it assumes options are restricted to scales and datalabels with a predetermined structure. Consider instead developing a function that combines the BarChart component's default options with any new options.

Comment (dimasciput) [reply django_project/frontend/src/containers/MainPage/Metrics/TotalCountPerActivity.tsx:104]: E.g. 

The original BarChart options :

```{ 'x' : 1, 'y': 2', 'z': {'a' : 2, 'b' : 3 }}```

A new component introduces additional options:

```{ 'z': {'c': 4, 'b': '5'}}```

Then the combined options will be :

```{ 'x' : 1, 'y': 2', 'z': {'a' : 2, 'b' : 5 , 'c':  4}}```

**Review by dimasciput — CHANGES_REQUESTED** [review]

File: django_project/frontend/src/containers/MainPage/Metrics/TotalCountPerActivity.tsx (hunk 1 above)
Comment (zamuzakki) [reply django_project/frontend/src/containers/MainPage/Metrics/TotalCountPerActivity.tsx:104]: @dimasciput I've updated it to use lodash merge
`const newOptions = _.merge(defaultOptions, options);`

Screenshot from 2023-11-14 09-47-04

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 7 min 1 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 6 min 9 sec

**Review by dimasciput — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 35 sec

**Pull request merged**